etl/steps/data/garden/hyde/2017/baseline.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
Would it make sense to include assertion that we mapped every country on line
```
population["country"] = population.country.apply(lambda c: country_mapping[c])
```
(i.e. adding check for null values)? No need to do it right now, might be useful to have a helper function for that.

Reply:
Ideally, we'd have a one-liner for the entire country-mapping-only harmonisation. But we can have a few ones like this that are simple for now.